skribenten-web/frontend/package.json

```diff
@@ -18,11 +18,14 @@
     "@navikt/aksel-icons": "5.9.2",
     "@navikt/ds-css": "5.9.2",
     "@navikt/ds-react": "5.9.2",
-    "@tanstack/react-location": "3.7.4",
     "@tanstack/react-query": "5.8.1",
```

Comment:
kanskje lage variabler for versjoner som henger sammen som react-query og react-query-devtools. Evt bump til 5.8.4 i samme slengen.

Reply:
Det er ikke given at query og tilhørende devtools er på samme versjon. Tipper de majorbumpes samtidig pga breaking changes, men ellers kan de kjøre eget løp